File: frontend/src/types/userProfile.ts
export interface UserProfile {
  operation_type: string
  marketplace: string
  niche: string
  goal: string
  investment_range: string
  experience_level: string
  traffic_type: string
}

export const defaultProfile: UserProfile = {
  operation_type: 'Vendedor',
  marketplace: 'Shopee',
  niche: 'Tecnologia',
  goal: 'Maior lucro',
  investment_range: 'R$ 500 a R$ 2.000',
  experience_level: 'Iniciante',
  traffic_type: 'Marketplace',
}

File: backend/app/models/user_profile.py
from __future__ import annotations

from sqlalchemy import Integer, String
from sqlalchemy.orm import Mapped, mapped_column

from app.database.db import Base


class UserProfileORM(Base):
    __tablename__ = "user_profiles"

    id: Mapped[int] = mapped_column(Integer, primary_key=True, autoincrement=True)
    operation_type: Mapped[str] = mapped_column(String(80))
    marketplace: Mapped[str] = mapped_column(String(80))
    niche: Mapped[str] = mapped_column(String(80))
    goal: Mapped[str] = mapped_column(String(120))
    investment_range: Mapped[str] = mapped_column(String(80))
    experience_level: Mapped[str] = mapped_column(String(80))
    traffic_type: Mapped[str] = mapped_column(String(80))

